src/core/tests/type_prop/eye.cpp

```diff
@@ -0,0 +1,276 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "ngraph/opsets/opset9.hpp"
+#include "type_prop.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+TEST(type_prop, eye_constant) {
+    auto num_rows = op::v0::Constant::create(element::i64, Shape{}, {6});
+    auto num_columns = op::v0::Constant::create(element::i64, Shape{}, {3});
+    auto diagonal_index = op::v0::Constant::create(element::i64, Shape{}, {0});
+
+    auto eye = std::make_shared<op::v9::Eye>(num_rows, num_columns, diagonal_index, element::bf16);
+
+    EXPECT_EQ(eye->get_output_element_type(0), element::bf16);
+    EXPECT_TRUE(eye->get_output_partial_shape(0).same_scheme(PartialShape{6, 3}));
```

Comment:
Please follow this approach in all these tests
```suggestion
    EXPECT_EQ(eye->get_output_partial_shape(0), PartialShape{6, 3});
```

Reply:
Changed everywhere. Thanks